crates/bevy_pbr/src/light.rs

```diff
@@ -1,64 +1,46 @@
 use bevy_core::Byteable;
 use bevy_ecs::reflect::ReflectComponent;
 use bevy_reflect::Reflect;
-use bevy_render::{
-    camera::{CameraProjection, PerspectiveProjection},
-    color::Color,
-};
+use bevy_render::color::Color;
 use bevy_transform::components::GlobalTransform;
-use std::ops::Range;
 
 /// A point light
 #[derive(Debug, Reflect)]
 #[reflect(Component)]
-pub struct Light {
+pub struct PointLight {
     pub color: Color,
-    pub fov: f32,
-    pub depth: Range<f32>,
     pub intensity: f32,
     pub range: f32,
 }
 
-impl Default for Light {
+impl Default for PointLight {
     fn default() -> Self {
-        Light {
+        PointLight {
             color: Color::rgb(1.0, 1.0, 1.0),
-            depth: 0.1..50.0,
-            fov: f32::to_radians(60.0),
-            intensity: 200.0,
+            intensity: 100.0,
```

Comment:
I see you are changing the default intensity value from 200.0 to 100.0. Why?

Also, do examples that don't change the default still render OK?

Reply:
Hmm, not on purpose. I'll revert that.